README.md

```diff
@@ -2,24 +2,53 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+# Directory structure
+
+```
+task-manager-app/
+	tsconfig.json
+    src/
+      app.tsx
+      index.tsx
+      interfaces.ts
+      enum.ts
+      constants.ts
+      global.styles.ts
+      server/
+        db.json
+      components/
+        signInForm
```

Comment:
create naming for files in following structure:
sign-in-form/
sign-in-form.tsx